mmdeploy/mmcv/ops/nms_rotated.py

```diff
@@ -149,3 +149,153 @@ def symbolic(g,
             is_normalized_i=False,
             clip_boxes_i=False,
             outputs=2)
+
+
+class ONNXBEVNMSOp(torch.autograd.Function):
```

Comment:
It seems that this autograd Function is almost the same as ONNXNMSRotatedOp.
Can we just alias it as `ONNXBEVNMSOp = ONNXNMSRotatedOp`?

Reply:
Yes，there is a bug in ONNXNMSRotatedOp when i implementation fcos3d. But I found it fix in [fix mmroate (](https://github.com/open-mmlab/mmdeploy/commit/a6ed7eddc565e5e5f7923396acfd352532546c78)https://github.com/open-mmlab/mmdeploy/pull/976[)](https://github.com/open-mmlab/mmdeploy/commit/a6ed7eddc565e5e5f7923396acfd352532546c78) . So I will reuse ONNXNMSRotatedOp.